tests/test_cli.py

```diff
@@ -21,6 +21,11 @@ def setUp(self):
 
         self.botoClientMock = mockBotoClient({"ec2": self.ec2Client, "servicediscovery": self.sdClient})
 
+        try:
+            lastReconcileTimestampMetric.remove("srv-1")
+            upMetric.remove("srv-1")
+        except KeyError as e:
+            pass
```

Comment:
I have to do this because previous executions of tests were putting data in the metrics and causing false negatives. I realized of this when adding the new test  `testMainShouldGracefullyHandleAnErrorWhileCallingAwsAPI` began to fail:

```
..F.
======================================================================
FAIL: testMainShouldGracefullyHandleAnErrorWhileCallingAwsAPI (tests.test_cli.TestCli)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/app/tests/test_cli.py", line 71, in testMainShouldGracefullyHandleAnErrorWhileCallingAwsAPI
    self.assertIsNone(prometheusDefaultRegistry.get_sample_value("aws_cloud_unmap_last_reconcile_success_timestamp_seconds", labels={"service_id": "srv-1"}))
AssertionError: 1554372785.0 is not None

----------------------------------------------------------------------
Ran 4 tests in 0.183s

FAILED (failures=1)
```